src/globus_sdk/services/groups/data.py

```diff
@@ -263,7 +263,9 @@ def __init__(
         group_members_visibility: _GROUP_MEMBER_VISIBILITY_T,
         join_requests: bool,
         signup_fields: t.Iterable[_GROUP_REQUIRED_SIGNUP_FIELDS_T],
-        authentication_assurance_timeout: int | None = None,
+        authentication_assurance_timeout: int
+        | None
+        | utils.MissingType = utils.MISSING,
```

Comment:
Oof. This style runs afoul of my similar preference for implicit string concatenation: in parentheses. I'm surprised that black doesn't modify this already, but I may have to adjust to this style.

Reply:
It looks like `black` leaves this in the "user specified style".
I'll change this one -- and BTW I have a much stronger feeling that parenthesized is better here than with the strings.

I'll look into adding a `slyp` check for this later.